src/main/java/seedu/command/ViewCommand.java

```diff
@@ -18,8 +18,8 @@ public void execute() {
         try {
             Contact viewingContact = contactList.getContactAtIndex(index);
             TextUi.viewContactMessage(viewingContact, index);
-        } catch (IndexOutOfBoundsException | NullPointerException e) {
-            TextUi.numOutOfRangeMessage(contactList.getListSize() - 1);
```

Comment:
Good catch that `contactList.getListSize() - 1` will display `-1` on `TextUi`. 

cc: @ashrafjfr as this is also pertaining to your implementation.